src/SprykerMiddleware/Zed/Process/Dependency/External/ProcessToSymfonyEncoderAdapter.php

```diff
@@ -27,9 +27,9 @@ public function __construct()
      * @param string $format
      * @param array $context
      *
-     * @return string
+     * @return mixed
```

Comment:
why? should only be string, right?

Reply:
Symfony encoder returns  * @return string|int|float|bool, so phpstan said I should change it